### PR TITLE
add Week 9 Part time schedule

### PR DESCRIPTION
**Pull request by tvpeter**

### CODE REVIEW

File: part-time-schedule/week-9.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Primary focus
+
+This week the primary focus will be on Open Source Projects.
+
+### Monday
+
+We will start the week by having a Contributing Etiquette Workshop that will prepare us on how to 
+contribute to Open Source Projects. It will also look at polishing our Projects on Github 
+covering Readme, Licence, Contribution guidelines amongst others.
```
Comment (dunxen) [inline part-time-schedule/week-9.md:11]: I think we expected to have them done with the GitHub and project polishing with the extra time they received last week.

Comment (tvpeter) [reply part-time-schedule/week-9.md:11]: yes, most of them did. Not sure it's all of them. I will take it out then.

File: part-time-schedule/week-9.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+##### Design Document
+
+After the workshop, you are expected to identify a project to work on alongside specific issues, design 
+and submit a Document for approval before proceeding to start making contributions.
```
Comment (dunxen) [inline part-time-schedule/week-9.md:16]:
```suggestion
After the workshop, you are expected to identify a project to work on alongside specific issues, design 
and submit a document for approval describing the issue and why you want to work on it and how you feel that your skills are suited to it.

You must complete the design document and have it approved *before* proceeding with any contributions. 
```

**Comment by dunxen** [comment]
Thanks Peter, happy for you to merge after the changes are made :smile:

**Pull request merged**